.gitignore

```diff
@@ -19,6 +19,7 @@ lib
 dist
 bin
 artifacts
+tmp
```

Comment:
Currently, we ignore all `.csv` files.
So I think we don't need to ignore `tmp`.

It's better to run cli-kintone on the isolated directory in the future.